packages/svelte/src/internal/client/dom/elements/bindings/select.js

```diff
@@ -124,14 +137,12 @@ export function bind_select_value(select, get, set = get) {
 }
 
 /**
- * @template V
  * @param {HTMLSelectElement} select
- * @param {V} value
+ * @param {unknown[]} value
  */
 function select_options(select, value) {
 	for (var option of select.options) {
-		// @ts-ignore
-		option.selected = ~value.indexOf(get_option_value(option));
+		option.selected = value.includes(get_option_value(option));
```

Comment:
I'm pretty sure this has performance implications...I don't know why it was written this way tho

Reply:
If we want better (theoretical) performance, we should use neither `.indexOf` nor `.includes` (code  will be `O(select.options.length * value.length)`) but rewrite it as

```ts
const values = new Set(value);
for (var option of select.options) {
  option.selected = values.has(get_option_value(option));
}
```

which is now `O(select.options.length + value.length)`

We'll see what perf addicts say of this :eyes:

Reply:
Includes is fast! https://www.measurethat.net/Benchmarks/Show/6029/0/some-vs-filter-vs-indexof-vs-includes

Reply:
It might be the better choice... I'm just saying that it seems an odd change I would like whoever wrote that code opinion on 😁